test_mh_updater.cpp

```diff
@@ -0,0 +1,54 @@
+#include <math.h>
```

Comment:
Do we need this file? especially since it is in the root folder?

Reply:
This is @mberaha production, but I guess this is no more needed in the final version.

Reply:
I am moving this stuff out from the PR, this is clearly a test. We will be able to resume it at any time.